froide/publicbody/api_views.py

```diff
@@ -354,6 +355,7 @@ class Meta:
             "jurisdiction",
             "request_note_html",
             "geo",
+            "updated",
```

Comment:
Maybe for consistency the field should be called the same as for the others? So expose a different name than the database field name.